R/annotation-data-helper.R

```diff
@@ -10,9 +10,12 @@ NULL
 #' @importFrom GenomeInfoDb 'seqlevelsStyle<-'
 getTranscriptRanges <- function(txdb, species = 'Homo_sapiens') {
     # Retrieve transcript ranges with detailed metadata
```

Comment:
No major changes for `annotation-data-helper.R` 
- Shorten line width by spreading out code across more lines for most helper functions
- Add `parseTxDb` helper function